diff --git a/src/x11/loader_x11.c b/src/x11/loader_x11.c
index 338c705..9e651f9 100644
--- a/src/x11/loader_x11.c
+++ b/src/x11/loader_x11.c
@@ -70,13 +70,6 @@ x11_dri3_open(xcb_connection_t *conn,
    fixes_cookie = xcb_xfixes_query_version(conn,
                                            XCB_XFIXES_MAJOR_VERSION,
                                            XCB_XFIXES_MINOR_VERSION);
-   fixes_reply = xcb_xfixes_query_version_reply(conn, fixes_cookie, NULL);
-   if (fixes_reply->major_version < 2) {
-      close(fd);
-      fd = -1;
-   }
-   free(fixes_reply);
-
    return fd;
 }
 
